style.css

```diff
@@ -174,3 +178,307 @@ body {
   flex: none;
   flex-grow: 0;
 }
+
+/* Recent section */
+#recent-work {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  gap: 58px;
+  position: absolute;
+  width: 379px;
+  height: 3283px;
+  left: 0;
+  top: 708px;
+}
+
+#recent-work-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 24px;
+  width: 327px;
+  height: 125px;
+  order: 0;
+}
+
+#recent-work-text h1 {
+  width: 327px;
+  height: 97px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  order: 0;
+}
+
+#recent-work-text img {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 24px;
+  order: 0;
+}
+
+#card-row {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4px 2px;
+  gap: 24px;
+  width: 331px;
+  height: 2972px;
+  order: 1;
+}
+
```

Comment:
- [ ] Kindly use `display:grid` instead of using `flexbox` to match the project requirements and make your code more efficient. you got this 👍